vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -60,6 +61,14 @@ module InlineParameterNameHints =
         >> Seq.map (fun location -> location.ArgumentRange) 
         >> Seq.contains range
 
+    let private getSourceTextAtRange 
+        (sourceText: SourceText)
+        (range: range) =
+
+        let line = $"{sourceText.Lines[range.Start.Line - 1]}"
```

Comment:
Why does it have to be an interpolated string? Won't just `.ToString()` work?

Reply:
It will. I just like interpolated strings more :) if you have a strong preference, I can change it

Reply:
No preference in general, it's just interpolation tends to generate quite sophisticated code. Should be fine with strings (I presume `Line` collection has list of strings?).

Reply:
Well, I updated it to ToString anyway